docs/commands/highlight.md

```diff
@@ -0,0 +1,93 @@
+## Command highlight ##
+
+This command sets up custom highlighting for user set strings.
+
+The syntax is as following:
+
+```
+highlight (list|del|add|clear|regex) [match] [color]
```

Comment:
I would list the `[match]` and `[color]` part only for the subcommands, where you wouldn't use `[]` around them because they'd be mandatory.